src/e2e-test/features/bigquery/sink/BigQueryToBigQuerySink.feature

```diff
@@ -195,3 +195,134 @@ Feature: BigQuery sink - Verification of BigQuery to BigQuery successful data tr
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
     Then Verify the partition table is created with partitioned on field "bqPartitionFieldTime"
+
+  @BQ_SOURCE_DATATYPE_TEST @BQ_EXISTING_SINK_TEST
+  Scenario: Validate successful record transfer from BigQuery source(existing table),source table here has more columns than BigQuery sink(existing table) with update button schema and truncate table to true.
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Source"
+    When Select plugin: "BigQuery" from the plugins list as: "Source"
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "BigQuery" from the plugins list as: "Sink"
+    Then Connect plugins: "BigQuery" and "BigQuery2" to establish connection
+    Then Navigate to the properties page of plugin: "BigQuery"
+    And Enter input plugin property: "referenceName" with value: "Reference"
```

Comment:
We should use "useConnection:"true" here as per the ITN description .

Reply:
Resolved. Scenarios are updated and use connection is used in all the scenarios